src/Transports/MassTransit.Azure.ServiceBus.Core/AzureServiceBusTransport/ServiceBusSendTransportContext.cs

```diff
@@ -31,7 +31,7 @@ public class ServiceBusSendTransportContext :
         }
 
         public override string EntityName { get; }
-        public override string ActivitySystem => "service-bus";
+        public override string ActivitySystem => "azure_servicebus";
```

Comment:
```suggestion
        public override string ActivitySystem => "servicebus";
```

Ref: https://github.com/open-telemetry/semantic-conventions/pull/572/files